CONTRIBUTING.md

```diff
@@ -0,0 +1,151 @@
+Contributing to Mypy
+====================
+
+Welcome!  Mypy is a community project that aims to work for a wide
+swath of Python users and Python codebases.  If you're trying Mypy on
+your Python code, your experience and what you can contribute is
+important to the project's success.
+
+
+Getting started, building, and testing
+--------------------------------------
+
+If you haven't already, take a look at the project's README file and
+the [Mypy documentation](http://mypy.readthedocs.org/en/latest/), and
+try adding type annotations to your file and type-checking it with Mypy.
+
+
+Discussion
+----------
+
+If you've run into behavior in Mypy you don't understand, or you're
+having trouble working out a good way to apply it to your code, or
+you've found a bug or would like a feature it doesn't have, we want to
+hear from you!
+
+Our main forum for discussion is the project's [GitHub issue
+tracker](https://github.com/python/mypy/issues).  This is the right
+place to start a discussion of any of the above or most any other
```

Comment:
That doesn't quite mean the same thing, and "most any" -- i.e., just about any topic though there are probably some exceptions -- is more what I mean.